constants/members.json

```diff
@@ -2406,11 +2448,53 @@
     },
     {
       "name": "Anish Aggarwal",
-      "role": "developer",
-      "term": 1229,
-      "teams": ["ccbc"],
+      "role": "project lead",
+      "term": 1231,
+      "teams": ["internal-tools"],
```

Comment:
the role should be `technical lead` as it is used for indexing in `utils.ts`. Right now, PL's and PM's are placed last due to wrong indexing. We need to replace with correct role names as `technical lead` and `product manager`.